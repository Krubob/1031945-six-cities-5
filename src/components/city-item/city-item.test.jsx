import React from "react";
import renderer from "react-test-renderer";
import CityItem from "./city-item";

describe(`CityItem snapshot`, () => {
  it(`Should CityItem render correctly when isActive={true}`, () => {
    const tree = renderer
    .create(
        <CityItem
          name = {`Paris`}
          isActive = {true}
          onCityClick={()=>{}}
        />
    )
    .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should CityItem render correctly when isActive={false}`, () => {
    const tree = renderer
    .create(
        <CityItem
          name = {`Paris`}
          isActive = {false}
          onCityClick={()=>{}}
        />
    )
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
