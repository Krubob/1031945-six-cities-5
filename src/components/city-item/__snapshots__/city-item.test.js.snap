// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityItem snapshot Should CityItem render correctly when isActive={false} 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;

exports[`CityItem snapshot Should CityItem render correctly when isActive={true} 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
